### PR TITLE
Bootcamp Merge Sort Implementation

### PR DESCRIPTION
**Pull request by tangsaidi**
Hello,

Here is my implementation of merge sort. Let me know if there are any problems.
Thank you

Best regards,
Saidi Tang

### CODE REVIEW

**Review by benjaminwinger — CHANGES_REQUESTED** [review]
Looks good in general. Only issue is a minor one with the test name that's breaking your test (see inline comment below).

**Comment by tangsaidi** [comment]
Renamed the test and added core modules back as requested 👍

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,8 +2,10 @@ cmake_minimum_required(VERSION 3.0)
 
 project(warg-cv-bootcamp C CXX) 
 
+set(MODULES merge)
```
Comment (benjaminwinger) [inline CMakeLists.txt]: This sets the modules variable, not updates it. If you want to add merge to the list, add it after core in the existing set call. Currently it's being immediately overwritten to only include core.

**Comment by tangsaidi** [comment]
Added both "merge" and "core" to "MODULES" variable

**Review by benjaminwinger — APPROVED** [review]

**Comment by tangsaidi** [comment]
Should I leave this pull request open or just close it? And what's my next step?

**Comment by benjaminwinger** [comment]
Lets just close it. I have in the past merged the bootcamps into branches, but that also begins to add some bloat to the bootcamp repo.

As to the next step, I've added you to Asana, so you can now take a look at the tasks that need to be done. You should probably start with a task marked beginner, as those require less familiarity with the existing code. If you want any suggestions as to what to choose, feel free to contact me on slack. You also should probably let me know before you start working on something just because some of the tasks are a little out of date.

**Pull request closed without merging**